config/routes.rb

```diff
@@ -25,6 +25,7 @@
 
   root to: 'welcome#index'
   patch '/projects/archive/:id' => 'projects#archive'
+  patch '/projects/:id/archive_user/:user_id' => 'projects#archive_user'
```

Comment:
why this is not in `api/v1` namespace? It returns just `head` and looks like api endpoint